static.json

```diff
@@ -8,10 +8,5 @@
     "/img/*": "/img/",
     "/js/*": "/js/",
     "/**": "index.html"
-  },
-  "proxies": {
```

Comment:
Please don't remove this proxy configuration. I have tested that it works correctly on the deployment. All we need to do is test/merge PR #35 for local development.